feature_engine/datetime/datetime.py

```diff
@@ -280,14 +309,21 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             )
 
         # create new features
-        for var in self.variables_:
+        if self.variables_ == ["index"]:
```

Comment:
If we extract features from the index, I am wondering whether we should just name the features as the datetime they represent? we are not creating features from multiple variables, so in principle, there would be no need to add the variable name at the front to distinguish them from anything else?